ACHP/CoaxialHX.py

```diff
@@ -20,6 +20,8 @@ def Update(self,**kwargs):
         self.A_r_wetted=pi*self.ID_i*self.L
         # Wetted area of the glycol side (not including outer tube)
         self.A_g_wetted=pi*self.OD_i*self.L
+        # total area of conduction
+        self.A_c_w=(pi/4) * (self.OD_i**2 - self.ID_i**2) * self.L

```

Comment:
Thank you for pointing this out, and check after me. I forget about the cylindrical coordinate thermal conduction.
It is updated now!